app/segments/drill/[id]/submission/input.js

```diff
@@ -515,7 +535,7 @@ export default function Input({ drillInfo, setToggleResult, setOutputData }) {
                   color={"#F24E1E"}
                 />
                 <Appbar.Content
-                  title={drillInfo.drillTitle}
+                  title={getCombinedDrillTitle(drillInfo)}
```

Comment:
Same issue as before here, happening on ios + andriod.
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/84f2c550-47ae-4991-947f-107a7da75b1c)

Reply:
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/961ed9f9-847c-42db-9a4a-7153238e70b4)

Reply:
So what should be displayed for the title? I don't think just subType because that won't be enough information

Reply:
also noticing there is no target being generated for some reason...

Reply:
I think if we just do the drill type then that would be fine, at this point the user has already seen the drill subtype twice so they should know what subtype they are in. additionally, we can look into adding something in the description modal to help with this if we think it is a concern